doc/api/process.md

```diff
@@ -638,6 +638,48 @@ process's [`ChildProcess.disconnect()`][].
 If the Node.js process was not spawned with an IPC channel,
 `process.disconnect()` will be `undefined`.
 
+## process.dlopen(module, filename[, flags])
+<!-- YAML
+added: v0.1.16
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/12794
+    description: Added support for the `flags` argument.
+-->
+
+* `module` {Object}
+* `filename` {string}
+* `flags` {integer}.
```

Comment:
Nit: Dot at the end is not required.

Reply:
Also, it would be better to specify the default value of this optional parameter, which is `kDefaultFlags`. Right now `crypto` module's `.md` has examples for this.

Reply:
Hm, the default value is platform specific.

Reply:
All the more important to document it I believe.